components/HomePage/Hero.js

```diff
@@ -39,8 +36,9 @@ const ActionButtons = styled.div`
     padding-bottom: 3rem;
   `};
 
-  ${below.med`
+  ${below.small`
     flex-direction: column;
+    align-items: stretch;
```

Comment:
Love this...would like to get it built in to the base LinkButton component if possible

Reply:
So this is the wrapper of the LinkButtons components, not the LinkButton itself. A few other places use a similar pattern, but I think we can avoid it completely by removing the linkButton wrapping div. I'm going to give that a shot.